go.mod

```diff
@@ -62,37 +62,38 @@ require (
 	github.com/ipld/go-car v0.3.2
 	github.com/ipld/go-car/v2 v2.1.1
 	github.com/ipld/go-codec-dagpb v1.3.0
-	github.com/ipld/go-ipld-prime v0.14.2
+	github.com/ipld/go-ipld-prime v0.14.1
```

Comment:
Was this intentional? Do we want to downgrade this?

Reply:
No, reverted. https://github.com/ipfs/go-ipfs/pull/8680/commits/2d3b3c6bca4067b35f09e466e8f28a8302bd7266#r845619647